frontend/viewer/lingui.config.ts

```diff
@@ -0,0 +1,17 @@
+﻿import {jstsExtractor, svelteExtractor} from 'svelte-i18n-lingui/extractor';
+import {formatter} from "@lingui/format-json";
+import {defineConfig} from "@lingui/cli";
+
+export default defineConfig({
+  format: formatter({style: "lingui"}),
+  locales: ['en', 'es', 'fr'],
+  sourceLocale: 'en',
+  catalogs: [
+    {
+      path: 'src/locales/{locale}',
+      include: ['src'],
+      exclude: ['src/locales']
```

Comment:
This confuses me. Can you explain why we're excluding precisely the thing I would think we need to include?

Reply:
the `path` specifies where the localization files live for the `include` sources. But we don't want to scan the localization files for text to localize

Reply:
Gotcha